src/test/java/org/takes/facets/hamcrest/HmRqHeaderTest.java

```diff
@@ -74,9 +80,60 @@ public void testsHeaderNotAvailable() throws Exception {
             ),
             new HmRqHeader(
                 Matchers.not(
-                    Matchers.hasEntry("host", "fake.org")
+                    new EntryMatcher<String, String>("host", "fake.org")
                 )
             )
         );
     }
+
+    /**
+     * HmRqHeader can test header name and value available.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void testsHeaderNameAndValueAvailable() throws Exception {
+        MatcherAssert.assertThat(
+            new RqWithHeader(new RqFake(), "header1: value1"),
+            new HmRqHeader("header1", "value1")
+        );
+    }
+
+    /**
+     * HmRqHeader can test header name and value not available.
```

Comment:
@bdragan can test `whether header name is available and value is not available.`